ci/scripts/trigger_publish_artifacts.sh

```diff
@@ -14,16 +14,25 @@ for image in baseos peer orderer ccenv tools ca javaenv nodeenv; do
     docker push "hyperledger-fabric.jfrog.io/fabric-${image}:amd64-latest"
 done
 
-cd "${GOPATH}/src/github.com/hyperledger/fabric/release/linux-amd64/"
-mkdir -p "config"
-cp ../../sampleconfig/*yaml "config"
-tar -czvf "hyperledger-fabric-linux-amd64-${RELEASE}.tar.gz" bin config
-curl -u"${ARTIFACTORY_USERNAME}":"${ARTIFACTORY_PASSWORD}" \
-     -T "./release/linux-amd64/hyperledger-fabric-linux-amd64-${RELEASE}.tar.gz" \
-     "https://hyperledger.jfrog.io/hyperledger/fabric-binaries/hyperledger-fabric-linux-amd64-${RELEASE}.tar.gz"
-
-cd "${GOPATH}/src/github.com/hyperledger/fabric-ca/release/linux-amd64"
-tar -czvf "hyperledger-fabric-ca-linux-amd64-${RELEASE}.tar.gz" bin
-curl -u"${ARTIFACTORY_USERNAME}":"${ARTIFACTORY_PASSWORD}" \
-     -T "./release/linux-amd64/hyperledger-fabric-ca-linux-amd64-${RELEASE}.tar.gz" \
-     "https://hyperledger.jfrog.io/hyperledger/fabric-binaries/hyperledger-fabric-ca-linux-amd64-${RELEASE}.tar.gz"
+
+for target in linux-amd64 darwin-amd64 windows-amd64; do
+    cd "${GOPATH}/src/github.com/hyperledger/fabric"
+    make "release/${target}"
```

Comment:
Do you mean to use "make", or "mkdir -p"?

Reply:
I am not sure why the response, which I received in email notification, is not visible here. I will mark this resolved. Here is brett's response:
They are correct, the make target to build a binary release in Fabric is on the form make release/linux-amd64 which then creates a subdirectory of fabric release/linux-amd64 and populates it with the binaries. As an aside, I did test both of these pipelines and they work, both pushed their respective 3 sets of binaries to artifactory (linux, mac and windows)